Code/Editor/NewLevelDialog.cpp

```diff
@@ -210,7 +211,26 @@ void CNewLevelDialog::OnLevelNameChange()
     UpdateData(true);
 
     // QRegExpValidator means the string will always be valid as long as it's not empty:
-    const bool valid = !m_level.isEmpty() && ValidateLevel();
+    bool valid = !m_level.isEmpty() && ValidateLevel();
+    if (valid)
+    {
+        QString strLevelPath = m_levelFolders + "/" + m_level + "/" + m_level + EditorUtils::LevelFile::GetDefaultFileExtension();
```

Comment:
A slightly cleaner way of doing this might be:
```cpp
QString strLevelPath = QString("%1/%2/%2%3)
    .arg(m_levelFolders,m_level,EditorUtils::LevelFile::GetDefaultFileExtension());
```

Reply:
I would also suggest using [QDir](https://doc.qt.io/qt-5/qdir.html#absoluteFilePath) for this to avoid any issues with path separators direction on non-Windows platforms
```
QDir levelDir(QString("%1/%2/").arg(m_levelFolders, m_level);
strLevelPath = levelDir.absoluteFilePath(m_level + EditorUtils::LevelFile::GetDefaultFileExtension());

Reply:
I've the made changes as per your suggestions.